### ISSUE TITLE
Data: DRKG and DBpedia query datasets?

### ISSUE THREAD

**Issue by migalkin** [Akirato/PERM-GaussianKG#2]
Hi, the repo does not have any links to the datasets used in the paper, esp. the new ones on DRKG and DBpedia. Do you have any plans sharing those?

**Comment by Akirato** [comment]
The DRKG and DBPedia datasets are constructed in the same way as the other datasets from links; [DBPedia](https://www.kaggle.com/danofer/dbpedia-classes) and [DRKG](https://github.com/gnn4dr/DRKG) Exact files are huge and hence, we couldn't share it on the repository.

**Comment by migalkin** [comment]
I see, but creating datasets, for instance, as done in [KGReasoning from Stanford](https://github.com/snap-stanford/KGReasoning/blob/main/create_queries.py), involves a lot of random sampling calls. This means the query datasets sampled from those original triples might very well be different (eg, sampled by you and sampled by me), which renders results in the paper hardly reproducible 😕 

There are many options to store larger datasets - Dropbox  / Google Drive / S3 buckets - most of them are free to use

**Comment by Akirato** [comment]
Yes. That makes sense. Let me check if I can get my dataset and put them in a dropbox link for everyone's use. Thanks a lot for the suggestion.
